src/bci_build/package/nodejs/README.md.j2

```diff
@@ -0,0 +1,54 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Node.js](https://nodejs.org/) is a free, open-source, cross-platform JavaScript run-time environment that lets developers write server-side applications and tools outside of a browser.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+
+CMD [ "node", "./server.js" ]
+```
+
+Build and run the container image:
+
+```ShellSession
+$ podman build -t my-node-app .
+$ podman run -it -p 3000:3000 --rm my-node-app
+```
+
+The example above assumes that there is a `package-lock.lock` file in the application directory.
+To generate a `package-lock.lock` file, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} npm i --package-lock-only
+```
+
+To run a single script inside a container, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} node script.js
+```
+
+## Additional tools
+
+The following additional tools are included in the image:
+
+- git-core
+
```

Comment:
I find this section is of questionable usefulness
```suggestion
```

Reply:
Yes, I had a similar feeling.

Reply:
imho we should only document it if we have a test that enforces git-core being available. 

I think generally it is a good idea to list the packages that are core and auxiliary installed but that requires a bit more plumbing in the dockerfile generator to not having this listed manually in the readmes.

Reply:
I removed this section.